.github/workflows/thank-you.yml

```
@@ -18,26 +18,26 @@ jobs:
         uses: actions/github-script@v6.3.3
         with:
           script: |
-            const thankyouNote = 'Your efforts have helped advance digital healthcare and TeleICU systems. :rocket: Thank you for taking the time out to make CARE better. We hope you continue to innovate and contribute; your impact is immense! :raised_hands:'
+            const thankyouNote = 'Your efforts have helped advance digital healthcare and TeleICU systems. :rocket: Thank you for taking the time out to make CARE better. We hope you continue to innovate and contribute; your impact is immense! :raised_hands:';
             const options = {
               issue_number: context.issue.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
-            }
+            };
 
-            const result = await github.rest.issues.get({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-            })
+            const { data : { assignees, user } } = await github.rest.issues.get(options);
 
-            const { assignees, user } = result.data
+            const uniqueUsers = new Set(
+              assignees.length > 0
+                ? assignees.map(u => u.login)
+                : [user.login]
+            );
```

So if assignees are there, this wouldn't mention the PR author if the PR author is not in assignees right? Also we could directly prefix with `@` from here itself right since we are anyways iterating through the assignees here.

```suggestion
            const taggedUsers = Array.from(new Set([...assignees.map(u => `@${u.login}`), `@${user.login}`]).join(', ');
```